src/lib/components/commons/SearchBar/SearchBar.styled.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { SearchIcon } from '../../icons/SearchIcon';
+import { SvgIcon } from '../../icons/SvgIcon';
+
+export const StyledSearchIcon = styled(SvgIcon)`
+  position: absolute;
+  left: 0;
+`;
+
+export const StyledSearchInput = styled.input`
```

Comment:
why not using the InputStyles ?

Reply:
i wanted to give it a costume look more like the quick-filter of mui